docs/howto.rst

```diff
@@ -75,3 +77,100 @@ For more complex crash sets, pass a search URL to generate the list::
     app@socorro:app$ socorro-cmd fetch_crashids --num=all --url="https://crash-stats.mozilla.org/search/?product=Sample&date=%3E%3D2019-05-07T22%3A00%3A00.000Z&date=%3C2019-05-07T23%3A00%3A00.000Z" > crashids
     app@socorro:app$ cat crashids | socorro-cmd reprocess
 
+
+Processing requests for PII access
+==================================
+
+People file bugs in Bugzilla asking to be granted access to PII.
+
+Process for handling those:
+
+1. Check to see if they're a Mozilla employee. If they aren't, then we can't
+   grant them access.
+
+2. Make sure they've logged into Crash Stats prod. If they have, there will be a
+   user account with their LDAP username.
+
+3. Look them up in phonebook and find their manager.
+
+4. Reply in the bug asking the reporter to agree to the memory dump access
+   agreement. Make sure to copy and paste the terms in the bug comments as well
+   as the url to where it exists. Tag the reporter with a needinfo.
+
+5. Reply in the bug asking th emanager to verify the reporter requires access to
+   PII on Crash Stats for their job. Tag the manager with a needinfo.
+
+Then wait for those needinfos to be filled. Once that's done:
+
+1. Log into Crash Stats.
+2. Go into the admin.
+3. Look up the user.
+4. Add the user to the "Hackers" group.
+
+Then reply in the bug something like this::
+
+    You have access to PII on Crash Stats. You might have to log out and log
+    back in again. Let us know if you have any problems!
+
+    Thank you!
+
+That's it!
+
+
+Deploys
+=======
+
+(As of May 16th, 2019)
+
+We have two environments: stage and prod.
+
+Stage autodeploys any time we land something in the master branch. You can see
+notifications on IRC or Slack. Ask for where it is these days.
+
+To deploy to prod:
+
+1. Check out the master branch and make sure you have the latest commits.
+2. Run ``scripts/deploy_bug.py`` to get the summary and description text for the
+   deploy bug.
+3. `Create a bug <https://bugzilla.mozilla.org/enter_bug.cgi?format=__standard__&product=Socorro&component=Infra>`_
```

Comment:
We didn't used to do deploy bugs, but we do now. It's a really handy record later on for knowing what went out, how we verified it, any curiosities in the deploy, etc. I've looked at deploy bugs a lot after the fact.

Plus because it's a bug, we can link people to it which has been handy, too.